tests/test_geoprocessing.py

```diff
@@ -3167,7 +3193,8 @@ def test_stitch_rasters(self):
         stitch_by_etch_target_path = os.path.join(
             self.workspace_dir, 'stitch_by_etch.tif')
         pygeoprocessing.numpy_array_to_raster(
-            numpy.full((256, 256), -1, dtype=numpy.float32), -1, (1, -1), (0, 0),
+            numpy.full((256, 256), -1, dtype=numpy.float32), -
+            1, (1, -1), (0, 0),

```

Comment:
Could you bump that `-` down to the next line for readability?

Reply:
sure! not sure why my autopep8 broke the line there